src/actions/index.js

```diff
@@ -1,23 +1,31 @@
-import {FETCH_PHOTOS_TO_MODERATE} from "./actionTypes";
+import {FETCH_PHOTOS_TO_MODERATE, FETCH_FEEDBACKS_TO_MODERATE} from "./actionTypes";
 import dbFirebase from "../dbFirebase";
 
-let unregisterPhotosToModerateObserver = undefined;
+let unregisterToModerateObserver = undefined;
 
-export const startFetchingPhotosToModerate = (dispatch) => async () => {
-  if (!unregisterPhotosToModerateObserver) {
+export const stopFetchingToModerate = () => {
+  if (unregisterToModerateObserver) {
+    unregisterToModerateObserver();
+  }
+  unregisterToModerateObserver = undefined;
+}
+
+export const startFetchingToModerate = (collection, dispatch) => async () => {
```

Comment:
not convinced that it a good idea to merge "photo moderation" with "feedback reading"

Reply:
In the time I was coding that I read DRY principle. Even in my point of view since I was in University I don't like to repeat the same code because it looks ugly and less professional. Less code make it easier and neat to read and understand. Another reason is code reuse, I have a function that I can use for both.